libbeat/filter/condition.go

```diff
@@ -11,247 +11,308 @@ import (
 	"github.com/elastic/beats/libbeat/logp"
 )
 
+type RangeValue struct {
+	gte *float64
+	gt  *float64
+	lte *float64
+	lt  *float64
+}
+
+type EqualsValue struct {
+	Int uint64
+	Str string
+}
+
 type Condition struct {
-	Equals   map[string]EqualsValue
-	Contains map[string]string
-	Regexp   map[string]*regexp.Regexp
-	Range    map[string]RangeValue
+	equals   map[string]EqualsValue
+	contains map[string]string
+	regexp   map[string]*regexp.Regexp
+	rangexp  map[string]RangeValue
```

Comment:
In the future I am planning to add `or`, `and` and `not`. For example, your case becomes:

```
or:
  equals:
    http.code: 200
  equals:
   http.code: 302
```